spec/acceptance/class_spec.rb

```diff
@@ -125,10 +125,11 @@ class { 'grafana':
 end
 
 describe 'grafana class with latest grafana version' do
-  context 'update to beta release' do
+  context 'update to beta release', unless: fact('os.family') == 'Debian' do
     it 'works idempotently with no errors' do
       case fact('os.family')
       when 'Debian'
+        # As of 2023-03-22, Debian beta packages are non functinal and disabled
```

Comment:
the whole case block doesnt make sense. we could also drop it.

Reply:
The code duplication is somewhat recent in the code history.

At first I wanted to revert the part that made both snippets identical, but the grafana repos are really missing packages, so even with explicit versions it would probably not work on the long term (only 2 beta packages are available in the Debian repo).
My guess was that this is currently broken on the Grafana side and might be fixed in the future, and maybe we will want to test it again when it is fixed.  For this reason I just drop that comment and did not touched the rest.

If we assume we do not care about beta releases on Debian, we can simplify this, but I am not a heavy user of Grafana so can't tell if it is likely to be fixed one day or not.

I am fine with both options and don't have an opinion, so will follow what other people have to say in regard to this.